### DISCUSSION TITLE
Adding a hint to the index page that ASA Stats supports .algo names

### DISCUSSION THREAD

**Discussion by ipaleka** [asastats/channel#122] [General]
By @Motuwagon in [Discord](https://discord.com/channels/906917846754418770/910524302531649566/982991189416501288): 

- - - - -
Can it be mentioned at the when moon screen too
" enter wallet address or  .algo  wallet name"

- - - - -

This is a great suggestion! The above implies changing the existing "Enter Algorand address" placeholder hint:
![hint](https://user-images.githubusercontent.com/49662536/172053199-a5d77946-90b5-4db7-a1f4-e6ddd8acfcc3.png)

Someone can argue does *.algo* name represent a synonym for "Algorand address", and also introducing *wallet* can add up to users' unnecessary contemplation.

At the same time, we have developed a random slogan functionality and we use it to pick between only two options:
- "All your Algorand assets on one dashboard"
- "Your entire Algorand portfolio in one place"

Wouldn't be great to solve both issues by adding some more slogans?

**Comment by moloch10** [comment]
> This is a great suggestion! The above implies changing the existing "Enter Algorand address" placeholder hint: ![hint](https://user-images.githubusercontent.com/49662536/172053199-a5d77946-90b5-4db7-a1f4-e6ddd8acfcc3.png)

How about changing the placeholder to something like:

Enter .algo / Algorand address

**Reply by ipaleka** [reply]
> How about changing the placeholder to something like:
> 
> Enter .algo / Algorand address

As I said, that brings more unneeded contemplation for our users as Algorand addresses and .algo names are synonyms, and we'd add a bit of difference with that construction and another bit of too much importance for .algo names.

Afaic adding a few slogans that have .algo in them will solve this.
